services/session.go

```diff
@@ -91,18 +92,33 @@ func CloseSession(s *Session) error {
 	return nil
 }
 
+func getExpiryHours() int {
```

Comment:
This function should return a time.Duration instead of `int`. So we can set env variables to things like 1h or 20m. We can use the https://golang.org/pkg/time/#ParseDuration function for that.